app/src/main/AndroidManifest.xml

```diff
@@ -17,6 +17,15 @@
                 <category android:name="android.intent.category.LAUNCHER"/>
             </intent-filter>
         </activity>
+<<<<<<< HEAD
```

Comment:
It's probably a good idea to resolve these merge conflicts, rather than committing the annotations (which will break the code).